libraries/stdlib/genmdl/materialx/sampling.mdl

```diff
@@ -8,18 +8,17 @@ export const int MX_MAX_SAMPLE_COUNT = 49;
 // Size of all weights for all levels (including level 1)
 export const int MX_WEIGHT_ARRAY_SIZE = 84;
 
-// Q: How to implement this in MDL?
+// This is not available in MDL so just use a "small" number
```

Comment:
@ashwinbhat , @niklasharrysson : Let me know if you see a better alternative.
If it was textures only, I was thinking of jumping along a lower LOD of the texture. or taking a N/<texture-size> jump in texture space but I don't have LOD access nor texture size access.
Kai's suggesting of generating texture coordinate gradients using C++ code I don't believe will work as you can only acess the results in C++.